tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/tests/formatter/TclModelFormatterTest.xtend

```diff
@@ -0,0 +1,75 @@
+package org.testeditor.tcl.dsl.tests.formatter
+
+import org.junit.Test
+
+class TclModelFormatterTest extends AbstractTclFormatterTest {
```

Comment:
I think there's too much going on in the two test methods. I would rather split them and handle the various constructs individually.